engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/definition/validator/NotBlankValueValidatorExtractor.scala

```diff
@@ -0,0 +1,16 @@
+package pl.touk.nussknacker.engine.definition.validator
+
+import java.lang.reflect.Parameter
+
+import pl.touk.nussknacker.engine.api.definition.{NotBlankValueValidator, ParameterValidator}
+import pl.touk.nussknacker.engine.definition.validator.adnotation.NotBlank
+
+object NotBlankValueValidatorExtractor extends ValidatorExtractor {
```

Comment:
maybe we can have some base class for simple validator extraction? E.g. in validator you would define it's annotation?

Reply:
We will do it in future..

Reply:
why? It will be only ~5 LOC. In this change you wrote the same thing again so IMO in this change you should extract duplicated code. The third developer will think "Why should I do that if noone before didn't?"

Reply:
I've added `class AnnotationValidatorExtractor[AnnotationClass](parameterValidator: ParameterValidator) extends ValidatorExtractor `

Reply:
Good job, I've left one comment regards to this extractor